config-connector/solutions/networking/helm/mci/README.md

```diff
@@ -0,0 +1,153 @@
+# Multi-cluster ingress
+
+==================================================
+
+## NAME
+
+  multi-cluster ingress
+
+## SYNOPSIS
+
+Multi-cluster ingress solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the mci folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/networking/helm/mci
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector
+1. [Helm](../../../README.md#helm)
+1. Cloud Resource Manager API needs to be enabled on the project to use [ServiceUsage Resource](https://cloud.google.com/config-connector/docs/reference/resources#service). You can enable it by running:
+
+    ```bash
+    gcloud services enable cloudresourcemanager.googleapis.com --project [PROJECT_ID]
+    ```
+
+## USAGE
+
+All steps are run from this directory.
+
+1. Review and update the values in `./values.yaml`.
+
+1. Subsequent steps will use $PROJECT_ID variable
+
+    ```bash
+    export PROJECT_ID=[PROJECT_ID]
+    ```
+
+1. Configure clusters and load balancing resources with Helm:
```

Comment:
To match with the other Helm solutions, you can remove the `$` in the bash below, and add something like "where `PROJECT_ID` is the project you want to deploy the load balancer in."

Reply:
Fixed, no longer using $PROJECT_ID